README.md

```diff
@@ -37,25 +37,25 @@ Large Language Models (LLMs.) The "**lab**" in Instruct**Lab** 🐶 stands for
 
 ## ❓ What is `ilab`
 
-`ilab` is a Command-Line Interface (CLI) tool that allows you to:
+`ilab` is a Command-Line Interface (CLI) tool that allows you to perform the following actions:
 
 1. Download a pre-trained Large Language Model (LLM).
 1. Chat with the LLM.
 
-To add new knowledge and skills to the pre-trained LLM you have to add new information to the companion [taxonomy](https://github.com/instructlab/taxonomy.git) repository.
-After that is done, you can:
+To add new knowledge and skills to the pre-trained LLM, add information to the companion [taxonomy](https://github.com/instructlab/taxonomy.git) repository.
+
+After you have added knowledge and skills, you can perform the following actions:
```

Comment:
```suggestion
After you have added knowledge and skills to the taxonomy, you can perform the following actions:
```

Reply:
Hi @bjhargrave - thanks for the review!

Suggestion committed.